src/content/web/libraries.md

```diff
@@ -4,46 +4,41 @@ short-title: Web libraries
 description: Libraries and packages that can help you write Dart web apps.
 ---
 
-The [Dart SDK][] contains [dart:html][] and other libraries
-that provide low-level web APIs.
-You can supplement or replace these APIs using web packages.
+Dart provides several packages and libraries to support
+web app development, the recommended option being [package:web][].
+The [Dart SDK][] also contains other libraries that provide low-level web APIs.
 
-[Dart SDK]: /tools/sdk
-[dart:html]: {{site.dart-api}}/{{site.sdkInfo.channel}}/dart-html/dart-html-library.html
+## Web solutions
 
+[Migrate to package:web][migrate]
+: Learn how to migrate to `package:web`
+  from Dart's previous web solution, [dart:html][].
 
-## SDK libraries
+[package:web API reference][package:web]
+: Complete reference documentation for `package:web` on pub.dev.
```

Comment:
I wonder if it might be better to give a one-line summary of what this package is, instead of indicating that this link goes to it's documentation. Doing that for this and `js_interop` below might help a reader more quickly identify the right link for them without having to click on them all.

Reply:
That's a great idea, added!